mmcv/ops/active_rotated_filter.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext(
+    '_ext',
+    ['active_rotated_filter_forward', 'active_rotated_filter_backward'])
+
+
+class ActiveRotatedFilterFunction(Function):
+    """Encoding the orientation information and then produces orientation-
+    sensitive and orientation-invariant features to alleviate the inconsistency
```

Comment:
orientation-sensitive and orientation-invariant seem to have conflicts intuitively, do we have a more suitable and accurate description?

Reply:
You are right. It should be ``Encoding the orientation information and generating orientation-sensitive features``.  We generate ``orientation-invariant features`` by another op.